salt/modules/debbuild.py

```diff
@@ -571,12 +575,18 @@ def make_repo(repodir,
         salt '*' pkgbuild.make_repo /var/www/html
 
     '''
-    res = {'retcode': 1,
+    res = {
+            'retcode': 1,
             'stdout': '',
-            'stderr': 'initialization value'}
+            'stderr': 'initialization value'
+            }
+
+    retrc = 0
 
-    SIGN_PROMPT_RE = re.compile(r'Enter passphrase: ', re.M)
-    REPREPRO_SIGN_PROMPT_RE = re.compile(r'Passphrase: ', re.M)
+    if gnupghome:
+        if env is None:
```

Comment:
This might be cleaner if we AND the previous conditional.

Reply:
sure, had thought about doing that but wanted to match code in PR https://github.com/saltstack/salt/pull/47056, and clean it up later, but can do it now